webapp/templates/partials/scoreboard.html.twig

```diff
@@ -142,4 +142,23 @@
         Last Update: {{ null | printtime('%a %d %b %Y %T %Z') }}<br/>
         using <a href="https://www.domjudge.org/" target="_top">DOMjudge</a>
     </p>
+
+    <script>
```

Comment:
Could be in the extrafooter block?

Reply:
I think I have to copy the code to different files than (so double code) whereas this approach had only 1 place to make changes. But I will move the code.